examples/pip_parse/WORKSPACE

```diff
@@ -24,6 +24,7 @@ pip_parse(
     # can be passed
     # environment = {"HTTPS_PROXY": "http://my.proxy.fun/"},
     name = "pypi",
+    incompatible_generate_aliases = True,
```

Comment:
nit: Please wait for #1525 and remove this flag.

Reply:
Done